Maquetado/scss/styles.scss

```diff
@@ -41,9 +42,9 @@ body {
 }
 
 .tools-container {
-  display: flex;
-  justify-content: flex-start;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 250px 213px 200px;
```

Comment:
Actually, what you need is something like: 

```scss
grid-template-columns: repeat(3, max-content);
```

so that way, the columns will take the maximum width the element inside the cell allows it :smile: